packages/agoric-cli/README.md

```diff
@@ -42,14 +36,14 @@ Here is a simple script for how to get started.
 ```sh
 # Initialize your dapp project.
 # Note: Change the `demo` name to something meaningful.
-agoric init demo
+$AGORIC init demo
```

Comment:
`AGORIC` worked for me but `$AGORIC` didn't. Is `$` the command line prompt or intended to be part of `$AGORIC`?

Reply:
`$AGORIC` is a reference to the environment variable you defined above with the appropriate `export AGORIC=` statement.

Reply:
After entering `export AGORIC="$PWD/bin/agoric --sdk"`, I get the following in response to entering `$AGORIC`:

```
zsh: no such file or directory: /Users/katelynsills/code/agoric-sdk/packages/agoric-cli/bin/agoric --sdk
```

Is that expected?

Reply:
Michael changed agoric to an alias, which shouldn't have this issue.